### DISCUSSION TITLE
Create Mat from Byte[ ]

### DISCUSSION THREAD

**Discussion by onslauth** [emgucv/emgucv#734] [Q&A]
Hi,

Is it possible to create a `Mat` from a `Byte[ ]`?

Looking at the documentation,  it only seems to support creating an empty `Mat` with dimensions, or to use a `IntPtr` that points to the data? Specifically I am trying to convert from `NumSharp.NDArray` to `Mat`.

**Comment by sn4k3** [comment]
You can set Mat data from raw bytes but you need to init the mat with proper size before. `mat.SetTo(bytes);` It will copy bytes over, instance is not shared
